pkg/controller/grafana/grafana_controller.go

```diff
@@ -102,7 +102,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler, autodetectChannel chan sch
 					log.Error(err, fmt.Sprintf("error adding secondary watch for %v", common.RouteKind))
 				} else {
 					cfg.AddConfigItem(config.ConfigRouteWatch, true)
-					log.Info(fmt.Sprintf("added secondary watch for %v", common.RouteKind))
+					log.V(1).Info("added secondary watch for %v", common.RouteKind)
```

Comment:
@pb82 should we keep format strings for now? ideally I'd like to move to a mapped approach to logging in the future, I can revert back to formatted strings for now though

Reply:
No need to keep fmt.Sprintf, we can use the native approach of the logger